GameOfLife/GameOfLife/Game/BoardGame.cs

```diff
@@ -0,0 +1,65 @@
+﻿using GameOfLife.Utils;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// Implementation for games that use Board interface.
+    /// </summary>
+    public class BoardGame : Game<IBoard>
+    {
+        /// <summary>
+        /// Board of the game.
+        /// </summary>
+        private IBoard _gameBoard { get; set; }
+
+        /// <summary>
+        /// Constructor for the game.
+        /// </summary>
+        public BoardGame(IBoard gameBoard) => _gameBoard = gameBoard;
+
+        /// <inheritdoc/>
+        public override void Play()
+        {
+            while (State == GameState.Playing)
+            {
+                Action keyActions = () =>
+                {
+                    switch (Panel.GetKeyInput())
+                    {
+                        case ConsoleKey.Spacebar:
+                            Pause(); 
+                            Console.Clear();
+                            break;
+
```

Comment:
Empty line not needed

Reply:
True, that is why it has been removed. ;)